app/ide-desktop/lib/client/src/config.ts

```diff
@@ -0,0 +1,595 @@
+                            `synchronizes the refresh rate and frame rate of the monitor to ` +
+                            `ensure optimal picture quality, particularly in gaming scenarios. ` +
+                            `However, in applications that heavily rely on a graphical user ` +
+                            `interface, the utilization of VSync is not deemed essential. By ` +
+                            `disabling this feature, performance may be improved on hardware ` +
+                            `configurations with limited capabilities. In addition, disabling ` +
+                            `VSync also has the potential to reduce rendering latency. For a ` +
+                            `comprehensive understanding of this aspect, please refer to this ` +
+                            `thread: https://bugs.chromium.org/p/chromium/issues/detail?id=460919.`,
+                    }),
+                    disableSmoothScrolling: new content.Option({
+                        passToWebApplication: false,
+                        primary: false,
+                        default: true,
+                        description:
+                            `Disable smooth scrolling feature. This modification has the ` +
+                            `potential to reduce latency experienced with input devices. For ` +
+                            `further elaboration, please refer to this thread: ` +
+                            `https://news.ycombinator.com/item?id=28782493.`,
+                    }),
+                    enableNativeGpuMemoryBuffers: new content.Option({
+                        passToWebApplication: false,
+                        primary: false,
+                        default: true,
+                        description: `Enable native CPU-mappable GPU memory buffer support on Linux.`,
+                    }),
+                },
+            }),
+
+            profile: new content.Group({
+                options: {
+                    loadWorkflowProfile: new content.Option({
+                        passToWebApplication: false,
+                        default: [] as string[],
+                        description:
+                            `Load a performance profile. For use with developer tools such as ` +
+                            `the 'profiling-run-graph' entry point.`,
+                        primary: false,
+                    }),
+                    saveWorkflowProfile: new content.Option({
+                        passToWebApplication: false,
+                        default: '',
+                        description:
+                            `Record a performance profile and save it to a file. To view the ` +
+                            `results, use the 'profiling-run-graph' entry point, such as ` +
+                            `'enso -startup.entry-point=profiling-run-graph -profile.load-workflow-profile=profile.json'.`,
+                        primary: false,
+                    }),
```

Comment:
I would call these options `loadProfile` / `saveProfile`--they are not only used for automated profiling *workflows*, but also for interactive profiling. When the profile hotkey is pressed (Ctrl+Alt+P), a profile file is written to the path provided here.